waitsensor/waitsensor.go

```diff
@@ -301,9 +308,9 @@ func (cs *counter) countDets(dets []objdet.Detection) int {
 }
 
 // return the level, and the string associated with that level
-func (cs *counter) counts2class(count int) string {
+func counts2class(count float64, thresholds []Bin) string {
```

Comment:
no longer a method, did not want to pass around the "counter" struct to a lot of other structs unnecessarily